src/neo/SmartContract/CallFlags.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Neo.SmartContract
+{
+    [Flags]
+    public enum CallFlags : byte
+    {
+        None = 0,
+        AllowModifyStates = 1 << 0,
+        AllowCall = 1 << 1,
```

Comment:
It looks good to me, maybe @igormcoelho has another possible Flags.

I took a look at this comment here: https://github.com/neo-project/neo/issues/446#issuecomment-437294243

Reply:
Deny is blacklist, Allow it's a whitelist, usually i prefer to use white lists, what do you prefer?